code/__HELPERS/lists.dm

```diff
@@ -392,5 +392,18 @@
 		if(islist(.[i]))
 			.[i] = .(.[i])
 
+#if DM_VERSION > 512

```

Comment:
I built a timebomb to remind us to remove it when the fated day comes

it will also double as a tracker for lummox's failure to implement it.